qiskit_nature/second_q/algorithms/excited_states_solvers/excited_states_eigensolver.py

```diff
@@ -85,6 +85,13 @@ def get_qubit_operators(
 
         if aux_operators is not None:
             for name_aux, aux_op in aux_operators.items():
+                if isinstance(aux_op, PauliSumOp):
+                    warn_deprecated(
+                        "0.6.0",
+                        old_type=DeprecatedType.ARGUMENT,
+                        old_name="PauliSumOp",
+                        new_name="SparsePauliOp",
+                    )
```

Comment:
This deprecation warning comes out as

>DeprecationWarning: The PauliSumOp argument is deprecated as of version 0.6.0 and will be removed no sooner than 3 months after the release. Instead use the SparsePauliOp argument.

Basically we are deprecating passing objects of PauliSumOp type as aux_operators and instead requiring them to be of SparsePauliOp type. The message is more making me look to an argument named PauliSumOp, which it doesn't have

If I replace `argument` by `type`, in the original message, (not that we have a DeprecatedType enum to do that at present) it comes closer (as below) but lacks aux_operators argument to correlate the warning

DeprecationWarning: The PauliSumOp type is deprecated as of version 0.6.0 and will be removed no sooner than 3 months after the release. Instead use the SparsePauliOp type.